test/cli/autogen-multipkg-autoloader/autogen-multipkg-autoloader.sh

```diff
@@ -7,11 +7,12 @@ preamble="# frozen_string_literal: true
 
 rm -rf output
 mkdir -p output
+echo "TODO: fix this test once autogen-autoloader-packaged is supported"
 
 main/sorbet --silence-dev-message --stop-after=namer \
   --stripe-packages \
   -p autogen-autoloader:output \
-  --autogen-autoloader-modules={Foo,FooMethods,Bar,BarMethods} \
+  --autogen-autoloader-modules=Project \
```

Comment:
Does this change the intent of the original test?

Reply:
Not really, now all the packages have the same root. That said this test is disabled for now.

Reply:
See https://github.com/sorbet/sorbet/pull/4136/files#diff-bb011a52ccb3f4f207975974228d74b84e99c231b375ea892430861c76de4fffR591